merlin/models/tf/__init__.py

```diff
@@ -122,6 +122,20 @@
 from merlin.models.tf.prediction_tasks.multi import PredictionTasks
 from merlin.models.tf.prediction_tasks.regression import RegressionTask
 from merlin.models.tf.prediction_tasks.retrieval import ItemRetrievalTask
+from merlin.models.tf.transformers.block import (
```

Comment:
If the idea is to make transformers an optional dependency. We may not be able to add these imports to this top-level module in this way?

Reply:
Good catch! We should wrap this in a try/catch to handle the case where `transformers` is not installed.

Reply:
I added a dependency check `is_transformers_available()` to import the blocks only when `transformers` is installed.